cogs/reminder.py

```diff
@@ -140,7 +140,7 @@ async def reminddelete(self,
     async def repeat(self,
                      ctx: mido_utils.Context,
                      channel: Union[discord.TextChannel, str],
-                     interval: Union[mido_utils.Time, str],
+                     interval: Union[mido_utils.Time],
```

Comment:
union here is redundant, it just merges types https://docs.python.org/3/library/typing.html#typing.Union